wolfcrypt/test/test.c

```diff
@@ -391,7 +391,7 @@ int memory_test(void);
 #ifdef HAVE_VALGRIND
 int mp_test(void);
 #endif
-#ifdef WOLFSSL_PUBLIC_MP
+#if defined(WOLFSSL_PUBLIC_MP) && !defined(WOLFSSL_SP_MATH)
```

Comment:
Does it make sense to use  #if defined(WOLFSSL_PUBLIC_MP) && defined(WOLFSSL_KEY_GEN) instead?

Reply:
Perfect.